gusto/diagnostics.py

```diff
@@ -1687,6 +1687,110 @@ def setup(self, domain, state_fields):
         super().setup(domain, state_fields, vorticity_type="relative")
 
 
+class CompressibleVorticity(DiagnosticField):
+    u""" Base diagnostic Field for three dimensional Vorticity """
```

Comment:
does this (should this?) also work in a vertical slice?

Reply:
This is a good question, which im not sure on so will have a test around and decide if it does / should it, i think it should work but may require a bit of thinking

Reply:
Since you've put in effort to make this work in a vertical slice, you can change this comment!